generated-platform-project/quarkus-kogito/integration-tests/pom.xml

```diff
@@ -18,6 +18,7 @@
     <module>integration-tests-quarkus-processes-postgresql</module>
     <module>kogito-quarkus-integration-test-maven-devmode</module>
     <module>integration-tests-quarkus-decisions</module>
+    <module>integration-tests-quarkus-gradle</module>
```

Comment:
TBF this was not supposed to end up here (yet?) -- how do we exclude this @aloubyansky

Reply:
````
                                    <test>
                                        <excluded>true</excluded>
                                        <artifact>org.kie.kogito:integration-tests-quarkus-gradle:${kogito-quarkus.version}</artifact>
                                    </test>
````
Should do it.

Reply:
@evacchi done with the help of @aloubyansky